archlinux/PKGBUILD

```diff
@@ -21,21 +20,39 @@ backup=()
 options=()
 changelog=
 
-source=(PKGBUILD-z-qubes-session.sh)
+source=()
+md5sums=()
 
 noextract=()
-md5sums=() #generate with 'makepkg -g'
-pa_ver=$((pkg-config --modversion libpulse 2>/dev/null || echo 0.0) | cut -f 1 -d "-")
+pa_ver=16.1 #specify this manually please
+# If using pkg-config, you must update libpulse first.
+#pa_ver=$((pkg-config --modversion libpulse 2>/dev/null || echo 0.0) | cut -f 1 -d "-")
+
+export PA_VER_FULL=$pa_ver
+
+if [ -e archlinux ] # on no, at repo root
+then
+    echo Do not run 'makepkg' at repo root! >&2
+    cd archlinux
+fi
+
+pkgver=$(cat $startdir/../version)
+
+clean() {
+    rm $srcdir/* -r || true
+}
 
 build() {
 
-for source in Makefile appvm-scripts gui-common include pulse gui-agent common xf86-input-mfndev xf86-video-dummy xf86-qubes-common window-icon-updater version; do
-  (ln -s $srcdir/../$source $srcdir/$source)
+rm $srcdir/PKGBUILD-z-qubes-session.sh || true
+cp $startdir/PKGBUILD-z-qubes-session.sh $srcdir/
+for source in Makefile appvm-scripts gui-common include pulse gui-agent common xf86-input-mfndev xf86-video-dummy xf86-qubes-common window-icon-updater version; do    
+  rm $source -r || true
```

Comment:
`rm -r` here too. And avoid `|| true` - if there is some specific error to ignore, prevent it from happening (I guess you meant `No such file or directory` here -> `if [ -e "$source" ]; then rm -r "$source"; fi`